.clang-tidy

```diff
@@ -116,5 +121,11 @@ CheckOptions:
     value:           'false'
   - key:             google-readability-function-size.StatementThreshold
     value:           '800'
+  - key:             cppcoreguidelines-narrowing-conversions.WarnOnIntegerToFloatingPointNarrowingConversion
+    value:           'true'
```

Comment:
The docs say this is true by default. Why is it necessary to explicitly set this?

https://releases.llvm.org/16.0.0/tools/clang/tools/extra/docs/clang-tidy/checks/cppcoreguidelines/narrowing-conversions.html

Reply:
I suppose it's not, unless the default is different in a different version

Reply:
If we're unsure, please remove it and open an issue to investigate it further.

Reply:
Removed. I think I added it because I saw it in the release notes and didn't check what the default was.